### PR TITLE
feat: add Jose to the website

### PR DESCRIPTION
**Pull request by pneumatick**
Added most of my profile information in people.yml; Added main picture (waiting on hover).

Note: This branch is not currently ready to merge.
Missing information: brown_directory_uuid, hover picture

### CODE REVIEW

**Comment by galenwinsor** [comment]
Just request someone's review when you're ready!

**Review by mzhan157 — APPROVED** [review]
looks good to me!

File: about/people.yml [hunk 1]
```diff
@@ -439,3 +439,12 @@ data:
     brown_directory_uuid: 542a2511-9c28-4f0d-a20b-107e002929fb
     bio: Melinda is a summer research intern working with the Computational Biology Core at CCV. She is an undergraduate student at Brown concentrating in Computational Biology. She loves matcha lattes and long walks in Providence.
     image: melinda_main.jpg
+    name: Jose Harvey
+    type: Part Time
```
Comment (eldu) [inline about/people.yml:443]: I believe you need to add in the hyphen so it knows it's a separate entry. Right now you've overridded Melinda's entry 

```suggestion
  - name: Jose Harvey
    type: Part Time
```

Comment (eldu) [reply about/people.yml:443]: Check out a YAML tutorial about arrays

File: images/people/jose_hover.jpg
Comment (eldu) [inline images/people/jose_hover.jpg:1]: The main and hover images have different extensions `.jpg` vs `.jpeg`. Unify that and I think it'll fix the hover

**Review by eldu — CHANGES_REQUESTED** [review]
Few things to change. Nice images !

**Review by eldu — CHANGES_REQUESTED** [review]
This looks good! Please merge main into your branch to resolve any merge conflicts

File: about/people.yml [hunk 2]
```diff
@@ -474,6 +474,15 @@ data:
...
     bio: Jesus is a data science intern working with the High Performance Computing division at CCV. He is a rising junior at Brown University studying Computer Science. His academic interests include software development, machine learning, and mathematics. Outside of work, you can find him playing guitar, learning Vietnamese, or listening to music.
     image: jesus_main.jpg
+  - name: Jose Harvey
+    type: Part Time
+    team: Advanced Research Computing
+    subteam: Graphics, Software, and Data Core
+    title: Data Science Intern
+    github_username: pneumatick
+    brown_directory_uuid: aceb7ec3-763c-4179-83b2-96417dbacf9f 
+    bio: Jose is a data science intern in the Advanced Reseach Computing division at CCV. He is an undergraduate student at the University of Rhode Island studying Computer Science. In his spare time, he enjoys reading, playing his guitar, drawing, and exploring new places.
```
Comment (eldu) [inline about/people.yml:484]: typo

```suggestion
    bio: Jose is a data science intern in the Advanced Research Computing division at CCV. He is an undergraduate student at the University of Rhode Island studying Computer Science. In his spare time, he enjoys reading, playing his guitar, drawing, and exploring new places. 
```

**Review by eldu — APPROVED** [review]
Looks good! One typo, but otherwise good to go!!

**Pull request merged**